modules/http_push.c

```diff
@@ -22,6 +22,7 @@
 
 #include <syslog.h>
 #include <curl.h>
+#include <openssl/hmac.h>
```

Comment:
Yeah, will look around if there's something appropriate and ask here if that's ok.